homework/Vladimir_Avseenkov/HomeWork0/ex1_SignComporator/SignComporatorTest.java

```diff
@@ -0,0 +1,42 @@
+package Vladimir_Avseenkov.HomeWork0.ex1_SignComporator;
+
+import com.sun.xml.internal.ws.api.message.ExceptionHasMessage;
+
+public class SignComporatorTest {
+    SignComporator signComporator = new SignComporator();
```

Comment:
Best practices in test are to run them independently. In this case for every method it's own signComporator should be used

Reply:
fixed